libs/shapelib/CMakeLists.txt

```diff
@@ -189,7 +189,7 @@ foreach(executable ${executables})
     get_target_property(${executable}_LOC ${executable} LOCATION)
     file(APPEND ${CMAKE_CURRENT_BINARY_DIR}/script.sed "s?\\./${executable}?${${executable}_LOC}?\n")
   endif(BUILD_TEST)
-  install(TARGETS ${executable} DESTINATION ${CMAKE_INSTALL_BINDIR})
+  # install(TARGETS ${executable} DESTINATION ${CMAKE_INSTALL_BINDIR})
```

Comment:
You just remove the line, there is no reason to comment it, that's why we use version control.

Reply:
Well, I was confused why the shapelib sources have been added as a copy instead of a submodule. Since I'm planning to put some effort into cleaning up this, too, I actually didn't care whether it looks messy ot not.
But yeah, far enough. I'm going to clean it up.

Reply:
https://github.com/mavlink/qgroundcontrol/pull/9569/commits/c82b10a477b282795120da0ef89032fd7edb0ad9